tests/entrypoints/openai/test_chat.py

```diff
@@ -250,11 +250,13 @@ async def test_more_than_one_prompt_logprobs_chat(
     [MODEL_NAME, "zephyr-lora"],
 )
 async def test_single_chat_session(client: openai.AsyncOpenAI, model_name: str):
+    # Finish reason may be "length" or "stop" on ROCm due to different tokenization
```

Comment:
Why would tokenization be different in the platforms?

I would suggest that instead, we either change the prompt or change the max_completion_tokens to be something smaller. This is going to be a PITA to maintain